Cargo.lock

```diff
@@ -2617,7 +2612,7 @@ dependencies = [
 [[package]]
 name = "jsonrpsee"
 version = "1.0.0"
-source = "git+https://github.com/svyatonik/jsonrpsee.git?branch=shared-client-in-rpc-api#99fb015dd9d678392dab8e90efdbd9b4a6b82fd9"
+source = "git+https://github.com/svyatonik/jsonrpsee.git?branch=shared-client-in-rpc-api#8e7fe826517a21c99e7868e056b793e5c6749fce"
```

Comment:
There's another small change in our fork of jsonrpsee. [This commit](https://github.com/svyatonik/jsonrpsee/commit/bc0d938217fd0c34e8dcaa08d034ab157c87b363) allows specifying using associated types in RPC methods. I.e.:
```rust
jsonrpsee::rpc_api! {
	pub(crate) Substrate<C: Chain> {
		#[rpc(method = "chain_getHeader", positional_params)]
		fn chain_get_header(block_hash: Option<C::Hash>) -> C::Header;
        }
}
```

Without it, I had to specify all used types explicitly, instead of using 'types container' (`Chain`).